src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/state/const.js

```diff
@@ -0,0 +1,6 @@
+//separated into different file because of a circular dependency caused otherwise
```

Comment:
could you rename the file to `states.js` ? I think it describes better the nature of these constants

Reply:
Maybe constants or types? I think states is also a bit misleading? As it does not contain any state just the action types. What do you think?

Reply:
good idea with `types.js`, this is how we called these files before we decided to merge them with actions.

Reply:
Will go with that then 👍